one-ui/src/test/java/com/marklogic/hub/curation/controllers/EnvironmentControllerTest.java

```diff
@@ -25,24 +25,9 @@
 import com.marklogic.hub.oneui.services.EnvironmentConfig;
 import com.marklogic.hub.oneui.services.EnvironmentService;
 import com.marklogic.hub.oneui.utils.TestLoggingAppender;
-import java.io.ByteArrayInputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.stream.Stream;
-import java.util.zip.ZipEntry;
-import java.util.zip.ZipInputStream;
 import org.apache.commons.lang3.StringUtils;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Could you change eclipse import setting and make sure that class count to use import with "*" is 30 or more.

<img width="985" alt="Screen Shot 2020-03-15 at 9 41 43 PM" src="https://user-images.githubusercontent.com/444927/76723820-df797d80-6705-11ea-839f-871ba8e72243.png">